api/HTMLInputElement.json

```
@@ -1155,60 +2625,59 @@
           }
         }
       },
-      "setRangeText": {
+      "useMap": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/setRangeText",
           "support": {
             "chrome": {
-              "version_added": "24"
+              "version_added": "1"
             },
             "chrome_android": {
-              "version_added": "25"
+              "version_added": "18"
             },
             "edge": {
-              "version_added": "79"
+              "version_added": "12"
             },
             "firefox": {
-              "version_added": "27"
+              "version_added": "1"
             },
             "firefox_android": {
-              "version_added": "27"
+              "version_added": "4"
             },
             "ie": {
-              "version_added": false
+              "version_added": "≤6"
             },
             "opera": {
-              "version_added": "15"
+              "version_added": "≤12.1"
             },
             "opera_android": {
-              "version_added": "14"
+              "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "6.1"
+              "version_added": "≤4"
             },
             "safari_ios": {
-              "version_added": "7"
+              "version_added": "≤3"
             },
             "samsunginternet_android": {
-              "version_added": "1.5"
+              "version_added": "1.0"
             },
             "webview_android": {
-              "version_added": true
+              "version_added": "1"
             }
           },
           "status": {
-            "deprecated": false,
             "experimental": false,
-            "standard_track": true
+            "standard_track": true,
+            "deprecated": false
```

```suggestion
            "deprecated": true
```

https://html.spec.whatwg.org/multipage/obsolete.html#HTMLInputElement-partial